src/nodeViews/createReactNodeViewConstructor.tsx

```diff
@@ -152,9 +160,10 @@ export function createReactNodeViewConstructor(
           decorations={decorations}
           isSelected={isSelected}
         >
-          {ContentDOMElementType && (
-            <ContentDOMElementType
-              // @ts-expect-error There are too many HTML tags, so typescript won't compute this union type
+          {childPortals}
+          {contentDOM && (
+            <div
+              style={{ display: "contents" }}
```

Comment:
Can we leave this change out of this PR and discuss it separately? I know you said outside GitHub that this might result in React warnings in some situations where the DOM hierarchy is otherwise invalid.

Reply:
Yeah; I'm going to go back to "use a span if contentDOM is inline otherwise use a div", then, I guess. The current behavior is simply wrong for lists (and the HTML5 spec does say that the only valid children of a `ul` is 0 or more `li`s, even if Firefox does actually render it correctly)